### PR TITLE
.build/kevm.rev: updated to latest master. Including extension to kprove.mak to support concrete-rules.txt

### PR DESCRIPTION
**Pull request by denis-bogdanas**

### CODE REVIEW

File: resources/kprove.mak [hunk 1]
```diff
@@ -94,14 +95,18 @@ ifneq ($(TIMEOUT),)
...
+
 KPROVE_PREFIX?=
 
 KPROVE_OPTS_java:=--deterministic-functions --cache-func-optimized --format-failures --boundary-cells k,pc \
 				  --log-cells k,output,statusCode,localMem,pc,gas,wordStack,callData,accounts,memoryUsed,\#pc,\#result
 KPROVE_OPTS_haskell:=
 
 KPROVE:=$(KPROVE_PREFIX) $(K_BIN)/kprove -v --debug -d $(KEVM_BUILD_DIR) -m VERIFICATION \
-        --z3-impl-timeout 500 $(SHUTDOWN_WAIT_TIME_OPT) $(TIMEOUT_OPT) \
+        --z3-impl-timeout 500 $(SHUTDOWN_WAIT_TIME_OPT) $(TIMEOUT_OPT) $(CONCRETE_RULES_OPT) \
```
Comment (daejunpark) [inline resources/kprove.mak]: update? https://github.com/runtimeverification/verified-smart-contracts/blob/master/deposit/bytecode-verification/Makefile#L14

File: deposit/bytecode-verification/.build/.kevm.rev [hunk 2]
```diff
@@ -1 +0,0 @@
-652576cb753656da51a25cea754d38c3d59470cd
```
Comment (daejunpark) [inline deposit/bytecode-verification/.build/.kevm.rev]: update? https://github.com/runtimeverification/verified-smart-contracts/blob/master/deposit/bytecode-verification/Makefile#L1

Comment (daejunpark) [reply deposit/bytecode-verification/.build/.kevm.rev]: BTW, please do not remove this local .kevm.rev file until the deposit verification is completed.  We are still frequently updating the kevm version.

Comment (denis-bogdanas) [reply deposit/bytecode-verification/.build/.kevm.rev]: Fixed all. Please review.

Comment (denis-bogdanas) [reply deposit/bytecode-verification/.build/.kevm.rev]: For Haskell support I guess I'll have to implement --concrete-rules in front-end.

Comment (denis-bogdanas) [reply deposit/bytecode-verification/.build/.kevm.rev]: Rolled back updates deposit/kevm.rev - some failures.

**Pull request merged**